src/post-message.ts

```diff
@@ -135,6 +135,20 @@ export type IncomingMessageMap = {
         id: string;
         payload: void;
     };
+    MATCH_CONTACT_DATA: {
+        id: string;
+        type: 'MATCH_CONTACT_DATA';
+        payload: {
+            matches: {
+                [key: string]: {
+                    name?: string;
+                    middleName?: string;
+                    surname?: string;
+                    encodedAvatar?: string;
```

Comment:
all attributes are optional?

Reply:
Yes, as I talked to @jmpuerta, all those attributes can come or not